### PR TITLE
feat(TUP-21518) Remove CryptoHelper

### PR DESCRIPTION
**Pull request by bhe-talendbj**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)
https://jira.talendforge.org/browse/TUP-21518
No behavior changes

**What is the new behavior?**
No behavior changes

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/dynamic/pref/AbstractDynamicDistributionPreference.java [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
     private IPreferenceStore prefStore;
 
-    private CryptoHelper cryptoHelper;
+    private StudioEncryption se = StudioEncryption.getStudioEncryption(StudioEncryption.EncryptionKeyName.SYSTEM);
```
Comment (coheigea) [inline main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/dynamic/pref/AbstractDynamicDistributionPreference.java]: Move into methods

File: main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/dynamic/pref/cdh/IDynamicCDHDistributionPreference.java [hunk 2]
```diff
@@ -38,5 +38,5 @@
 
     static final String PREF_PASSWORD = "distribution.dynamic.repository.cdh.password"; //$NON-NLS-1$
 
-    static final String PREF_PASSWORD_DEFAULT = "LycrWORKBL7x80mU8ziB6dCFh+ZjAYX/"; //$NON-NLS-1$
+    static final String PREF_PASSWORD_DEFAULT = PREF_USERNAME_DEFAULT; // $NON-NLS-1$
```
Comment (coheigea) [inline main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/dynamic/pref/cdh/IDynamicCDHDistributionPreference.java]: What does this change mean?

File: main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/dynamic/pref/hdp/IDynamicHDPDistributionPreference.java [hunk 3]
```diff
@@ -39,5 +39,5 @@
 
     static final String PREF_PASSWORD = "distribution.dynamic.repository.hdp.password"; //$NON-NLS-1$
 
-    static final String PREF_PASSWORD_DEFAULT = "LycrWORKBL7x80mU8ziB6dCFh+ZjAYX/"; //$NON-NLS-1$
+    static final String PREF_PASSWORD_DEFAULT = PREF_USERNAME_DEFAULT; // $NON-NLS-1$
```
Comment (coheigea) [inline main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/dynamic/pref/hdp/IDynamicHDPDistributionPreference.java]: What does this change mean?

File: main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/dynamic/pref/cdh/IDynamicCDHDistributionPreference.java (hunk 2 above)
Comment (bhe-talendbj) [reply main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/dynamic/pref/cdh/IDynamicCDHDistributionPreference.java]: We choose not to encrypt default password. The default password is clear text, the same as the default username.

File: main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/dynamic/pref/hdp/IDynamicHDPDistributionPreference.java (hunk 3 above)
Comment (bhe-talendbj) [reply main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/dynamic/pref/hdp/IDynamicHDPDistributionPreference.java]: We choose not to encrypt default password. The default password is clear text, the same as the default username.

File: main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/dynamic/pref/cdh/IDynamicCDHDistributionPreference.java (hunk 2 above)
Comment (nrousseau) [reply main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/dynamic/pref/cdh/IDynamicCDHDistributionPreference.java]: Just a note.. this is a user/password of nexus available even in the documentation.
Goal for nexus was not security, but only audit from IT to know where the login/download comes from

**Pull request merged**